config/level/shopware/shopware55.yaml

```diff
@@ -0,0 +1,18 @@
+services:
```

Comment:
Could you link some source of upgrade path?

E.g. link to `UPGRADE.md`, `changelog.md`, post on blog...

```yaml
# see: https://....file.md
services:
   ...
```

That way it's easier to find out more about the project and complete the list.

Reply:
I added the Upgrade.md for the developers. But the part with Magic Calls is currently not documented. We want to get rid of those calls in the future :)